btr-web/btr-main-app/utils/breadcrumbs.ts

```diff
@@ -1,18 +1,21 @@
 export function getBcrosHomeDashboardCrumb (): BreadcrumbI {
-  const { t } = useI18n()
+  const { $i18n } = useNuxtApp()
```

Comment:
I think this is a little nicer, but up to you:
`const t = useNuxtApp().$i18n.t`

Reply:
Updated the code to the nicer suggestion you have.